pkg/oc/cli/admin/registry/registry.go

```diff
@@ -484,6 +485,17 @@ func (opts *RegistryOptions) RunCmdRegistry() error {
 		case *kapi.Service:
 			t.Spec.SessionAffinity = kapi.ServiceAffinityClientIP
 			t.Spec.ClusterIP = clusterIP
+			if t.Annotations == nil {
+				t.Annotations = make(map[string]string)
+			}
+			t.Annotations["prometheus.io/path"] = "/extensions/v2/metrics"
+			t.Annotations["prometheus.io/port"] = "5000"
+			if tls {
+				t.Annotations["prometheus.io/scheme"] = "https"
+			} else {
+				t.Annotations["prometheus.io/scheme"] = "http"
+			}
+			t.Annotations["prometheus.io/scrape"] = "true"
```

Comment:
@dmage @legionus we'll want to make sure metrics are enabled in the registry when the operator creates/manages it.